filesystem/apps/editor/main.rs

```diff
@@ -1,20 +1,21 @@
 extern crate orbital;
 
-use std::*;
+use std::fs::File;
 
 use orbital::*;
+use std::io::*;
+use std::env;
 
 pub struct Editor {
     url: String,
     file: Option<File>,
     string: String,
-    offset: usize,
-    scroll_x: isize,
-    scroll_y: isize,
+    offset: u32,
```

Comment:
Why make offset u32, if it significantly increases the number of `offset as usize` statements?